dac161s997.h

```diff
@@ -22,10 +22,11 @@
 #define DAC161S997_H_
 
 /* Defines ********************************************************************/
-#define DAC161S997_MIN_NA   ((uint32_t)4000000)     /**< Min valid nA */
-#define DAC161S997_MAX_NA   ((uint32_t)20000000)    /**< Max valid nA */
-#define DAC161S997_LO_ALARM_NA ((uint32_t)2000000)  /**< Lo error value in nA */
-#define DAC161S997_HI_ALARM_NA ((uint32_t)22000000) /**< Hi error value in nA */
+#define DAC161S997_ALARM_LOW_FAIL_ERR   0x0100
+#define DAC161S997_ALARM_LOW_SAT_ERR    0x0200
+#define DAC161S997_ALARM_UNINIT_ERR    0x0400
+#define DAC161S997_ALARM_HI_SAT_ERR     0x0800
+#define DAC161S997_ALARM_HI_FAIL_ERR    0x1000
```

Comment:
Public macros should be documented
```suggestion
#define DAC161S997_ALARM_LOW_FAIL_ERR   0x0100 /**< Error flag for low device failure */
#define DAC161S997_ALARM_LOW_SAT_ERR    0x0200 /**< Error flag for high device failure */
#define DAC161S997_ALARM_UNINIT_ERR    0x0400 /**< Error flag for device uninitialized */
#define DAC161S997_ALARM_HI_SAT_ERR     0x0800  /**< Error flag for value set to upper bound saturation */
#define DAC161S997_ALARM_HI_FAIL_ERR    0x1000 /**< Error flag for value set to lower bound saturation */
```